collections/item.go

```diff
@@ -0,0 +1,48 @@
+package collections
+
+import (
+	"fmt"
+
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+)
+
+// NewItem instantiates a new Item instance, given the value encoder of the item V.
+func NewItem[V any](sk storetypes.StoreKey, prefix Prefix, valueEncoder ValueEncoder[V]) Item[V] {
+	return (Item[V])(NewMap[noKey, V](sk, prefix, noKey{}, valueEncoder))
+}
+
+// Item is a type declaration based on Map
+// with a non-existent key.
+type Item[V any] Map[noKey, V]
```

Comment:
I've also added Item. The implementation is super simple, if you feel it creates noise in the PR I can port it later.

Reply:
Is `Item` really a better name than `Singleton`? Or why not call it `Param` or `Var`?

Reply:
Nomenclature comes from cosmwasm storage plus so people jumping back and forth the two frameworks will find familiarity

Reply:
Item is a better name than singleton imo

Reply:
I kind of feel like intuitive naming is more important that cw storage plus similarity, but we can address in follow-ups